js/api-augment/definitions/schemas.ts

```diff
@@ -34,7 +34,7 @@ export default {
       model: "SchemaModel",
       model_type: "ModelType",
       payload_location: "PayloadLocation",
-      setting: "Vec<SchemaSetting>",
+      settings: "Vec<SchemaSetting>",
```

Comment:
this was the bug, rpc in node returns "settings" while api had "setting" causing fields mismatch